andela/wg-silicon
# [Feature #158626096] Cache nutritional values

### What does this PR do?
Caches nutritional plans


### Description of Task to be completed?
If a user has many plans, the overview renders slowly as too many DB-queries are fired just to calculate the total calories. They are cached to reduce the workload on the database.

### How should this be manually tested?
1. Log into wger, click on the `Nutrition` dropdown and select `Nutrition Plans`.
2. Add as many nutrition plans as possible.
3. Empty cache and hard reload and note the time the time taken for the nutrition plans to be returned.
4. Reload the tab normally and note the time the time taken for the nutrition plans to be returned.
5. Note that the second time takes a much shorter time that the first one.

### What are the relevant pivotal tracker stories?
- [Cache nutrition plans](https://www.pivotaltracker.com/story/show/158626096)

### Screenshots
__Without caching__
<img width="523" alt="screen shot 2018-07-11 at 11 47 35" src="https://user-images.githubusercontent.com/29580769/42561371-a18cde0a-8501-11e8-941f-a3edd25d2c56.png">

__With caching__
<img width="516" alt="screen shot 2018-07-11 at 11 48 05" src="https://user-images.githubusercontent.com/29580769/42561383-ac5d6868-8501-11e8-8105-1705eff04672.png">
